aptos-move/framework/aptos-stdlib/doc/bn254_algebra.md

```diff
@@ -0,0 +1,532 @@
+
```

Comment:
this file is derived from the comments in `bn254_algebra.move` and is currently out-of-date.
Each time you edit `bn254_algebra.move`, make sure you run `cargo run -p aptos-framework release` before committing.

Reply:
@nanne007